src/utils/format.ts

```diff
@@ -66,5 +67,17 @@ export function descaleBigIntToNumber(value: bigint | string, decimals: number):
 }
 
 export function formatAddress(address: string) {
```

Comment:
Can change to:
```suggestion
export function formatAddress(address: Address) {
```

Then we can remove the try-catch here